_sass/_features.scss

```diff
@@ -99,6 +99,31 @@ body.features-page {
 
 .feature {
     margin-bottom: $spacing-unit/2;
+
+    .feature-link {
+        margin-left: -$spacing-unit !important;
+    }
+}
+
+.feature, .feature-group {
+    .feature-anchor {
+        top: -($header-height + $header-border-top-height + $header-border-bottom-height + $spacing-unit/2);
```

Comment:
From what I remember when I was designing this, I'd just use this expression (looks more natural imho when I put this in inspector):
```sass
top: -($header-height + $spacing-unit);
```